docs/content/mobile/android/setup.md

```diff
@@ -3,9 +3,11 @@
 ## Get the dependencies for developing Android games
   1. Follow the steps in [Get the dependencies]({{urlRoot}}/setup-and-installing) and additionally install **Android build support** for Unity.
   1. Install Android prerequisites:
-    * [Android Studio](https://developer.android.com/studio/) -  Once installed, open Android Studio. Ensure to install:
-       * The Android SDK
-       * The Android Studio emulator
+    * [Android Studio](https://developer.android.com/studio/)
+       * At the Choose Components stage of the instalation, select **Android Virtual Device**.
+       * Once the instalation is complete, open Android Studio and select **Configure** > **SDK Manager**.<br>
+         Select the version you intend to develop your game for.<br>
+         Select **Apply** to download and install the matching **Android SDK** version.
```

Comment:
wfh today, so can't check how it renders. but do we not want different points / an enumeration for those two steps?

Reply:
good spot, fixed.